querydsl-sql/src/test/java/com/mysema/query/sql/dml/SQLInsertClauseTest.java

```diff
@@ -20,4 +20,17 @@ public void GetSQL() {
         assertEquals(ImmutableList.of(1), sql.getBindings());
     }
 
+    @Test
+    public void GetSQLWithPreservedColumnOrder() {
+        com.querydsl.sql.domain.QEmployee emp1 = new com.querydsl.sql.domain.QEmployee("emp1");
```

Comment:
Do you want me to update the fully qualified name or update the import of `QEmployee`?